### PR TITLE
Update limits-ediscovery20.md

### PR DESCRIPTION
**Pull request by msbemba**
Added age out details to the document.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8684

File: microsoft-365/compliance/limits-ediscovery20.md [hunk 1]
```diff
@@ -139,4 +139,6 @@ The limits described in this section are related to exporting documents out of a
...
 >
-> <sup>7</sup> This limit applies to downloading selected documents from a review set. It doesn't apply to exporting documents from a review set. For more information about downloading and exporting documents, see [Export case data in eDiscovery (Premium)](exporting-data-ediscover20.md).
+> <sup>7</sup> Age-Out period of the Azure Blobs that store eDiscovery Premium Collections is 1 year. Any collection created a year ago may no longer be accessible.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-ediscovery20.md]:
```suggestion
> <sup>7</sup> The age-out period of the Azure Blobs that store eDiscovery (Premium) collections is one year. Any collection created a year ago may no longer be accessible.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@v-tophillips Please merge this PR. Thanks

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**